src/pages/GameFinishPage/GameFinishPage.tsx

```diff
@@ -1,5 +1,40 @@
+import { DUMMY_DATA } from '../RankPage/RankData';
+import RankList from '../RankPage/RankList';
+
 const GameFinishPage = () => {
-  return <div>게임 피니시 페이지</div>;
+  return (
+    <div className='flex flex-col gap-[8rem] items-center'>
+      <section>
+        <span className='text-3xl font-bold font-[Giants-Inline]'>
+          3초 뒤 자동으로 로비로 이동합니다...
+        </span>
+      </section>
+      <section className='flex gap-[10rem] w-full'>
+        <div className='flex flex-1 justify-center items-end'>
+          <div className='bg-[#FFF4DC] h-[28rem] w-[14rem] flex items-center justify-center shadow text-5xl font-bold'>
```

Comment:
tailwind.config.js에 이미 #FFF4DC가 beige-100으로 정해져있으므로 이렇게 바꿔보는 건 어떨까용?! 🤔
```suggestion
          <div className='bg-beige-100 h-[28rem] w-[14rem] flex items-center justify-center shadow text-5xl font-bold'>
```

Reply:
tailwind.config.js에 beige-100으로 정해져있군요..! 배경색만 정해져있는줄 알았습니다